### ISSUE TITLE
exec /usr/local/bin/docker-entrypoint.sh: exec format error

### ISSUE THREAD

**Issue by Stooovie** [cnorick/ha-auto-guest-login#9]
Trying to install via Portainer on the same RPi4 that runs HA (via Portainer stack), I'm getting

`exec /usr/local/bin/docker-entrypoint.sh: exec format error
`
This is my docker compose file:

```
---
version: "3"
services:
  ha-auto-guest-login:
    image: norick/ha-auto-guest-login:main
    container_name: ha-auto-guest-login
    environment:
      - CLIENTID=http://192.168.0.181:8123/
      - URL=http://192.168.0.181:8123/
      - DASH=lovelace-mushroom
      - USER=host
      - PASS=aaaa
    ports:
      - 4077:80
    restart: unless-stopped
```

I tried to install the plain, dockerless version on a different RPi3 only to get this:

```
/home/pi/ha-auto-guest-login/server.js:1
import fetch from "node-fetch"
^^^^^

SyntaxError: Unexpected identifier
at Module._compile (internal/modules/cjs/loader.js:723:23)
at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
at Module.load (internal/modules/cjs/loader.js:653:32)
at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
at Function.Module._load (internal/modules/cjs/loader.js:585:3)
at Function.Module.runMain (internal/modules/cjs/loader.js:831:12)
at startup (internal/bootstrap/node.js:283:19)
at bootstrapNodeJSCore (internal/bootstrap/node.js:623:3)
```

Please help.

**Comment by stegno** [comment]
The same issue
<img width="452" alt="image" src="https://user-images.githubusercontent.com/38944977/203071797-a038d416-8e11-4c5e-9edf-0f56107fedaa.png">

**Comment by PromoFaux** [comment]
Just saw this issue after creating #10, the issue you are seeing is because the image is only built for `amd64` architecture.

In order to run on your Pi, just clone the repo, cd into the directory and run `docker build . --tag my-ha-auto-guest-login` and then replace the reference to `norick/ha-auto-guest-login:main` with `my-ha-auto-guest-login`, and it will use the locally built image

Or just wait for #10 if it is accepted.

**Comment by cnorick** [comment]
Fixed by #10
